coupons.go

```diff
@@ -3,27 +3,30 @@ package recurly
 import "encoding/xml"
 
 // Coupon represents an individual coupon on your site.
+// https://dev.recurly.com/docs/lookup-a-coupon
 type Coupon struct {
-	XMLName            xml.Name          `xml:"coupon"`
-	Code               string            `xml:"coupon_code"`
-	Name               string            `xml:"name"`
-	HostedDescription  string            `xml:"hosted_description,omitempty"`
-	InvoiceDescription string            `xml:"invoice_description,omitempty"`
-	State              string            `xml:"state,omitempty"`
-	DiscountType       string            `xml:"discount_type"`
-	DiscountPercent    int               `xml:"discount_percent,omitempty"`
-	DiscountInCents    int               `xml:"discount_in_cents,omitempty"`
-	RedeemByDate       NullTime          `xml:"redeem_by_date,omitempty"`
-	SingleUse          NullBool          `xml:"single_use,omitempty"`
-	AppliesForMonths   NullInt           `xml:"applies_for_months,omitempty"`
-	MaxRedemptions     NullInt           `xml:"max_redemptions,omitempty"`
-	AppliesToAllPlans  NullBool          `xml:"applies_to_all_plans,omitempty"`
-	CreatedAt          NullTime          `xml:"created_at,omitempty"`
-	PlanCodes          *[]CouponPlanCode `xml:"plan_codes>plan_code,omitempty"`
-}
-
-// CouponPlanCode holds an xml array of plan_code items that this coupon
-// will work with.
```

Comment:
Need to add the following fields: 
ID int 
CreatedAt NullTime 
UpdatedAt NullTime 
DeletedAt NullTime 

DiscountInCents should just be a NullInt. UnitAmount is for resources that return a map of currency, amount. 

Can you point me to `unique_coupon_codes_count` in the docs, I couldn't find it?

Reply:
It's in bulk coupon, also you can see the `DiscountInCents` is not a simple `int` type, but map of currency.

![screen shot 2018-07-27 at 1 23 53 pm](https://user-images.githubusercontent.com/1977400/43337347-9db2f5d8-91a1-11e8-8d60-e79d4a869e38.png)

Reply:
Ah got it, good catch. Thanks!